condicionales/numberParOImparA2.js

```diff
@@ -0,0 +1,20 @@
+const prompt = require("prompt-sync")();
+
+let evenNumber;
+let oddNumber;
+
+evenNumber = Number(prompt("Enter a even number: "));
+if (evenNumber % 2 !== 0) {
+	console.log(`You have not written an even number.`);
+	console.log(`Please run the program again to try again.`);
+}
+
+oddNumber = Number(prompt("Enter a odd number: "));
```

Comment:
This line is going to be executed even if the before number is not even. The program should end if the user enters a wrong number